config/default_config.yml

```diff
@@ -275,3 +275,57 @@ SCOREBOT:
       - WHAT_IS_A_BREAK
       - SURVIVORS
       - U_R_STILL_A_MAN
+
+
+# This allows you to subscribe any number of Discord webhooks and optional roles and/or users to mention
```

Comment:
We have a generic piece in the UI for that btw. Not sure how much work it'd be to reuse

Reply:
I think I'd prefer to leave it in `config.yml` for now as the settings UI is a bit cluttered, if we break the settings in the frontend out into multiple views in the future we could add it.